mobx/lib/add_todo_page.dart

```diff
@@ -3,6 +3,8 @@ import 'package:flutter_mobx/flutter_mobx.dart';
 import 'package:mobx_sample/model/todo.dart';
 
 class AddTodoPage extends StatefulWidget {
+  static const routeName = '/addTodo';
```

Comment:
All of the route-names are constants that can be kept separately. Let's not tie the route to a component this way. I suggest create a set of constants at the global level for the various routes or just have a single route object as a `Map<String, Widget>` somewhere. Please remove this tight coupling of route to components in all components